pkg/util/featuregates.go

```diff
@@ -0,0 +1,33 @@
+package util
+
+import (
+	"strings"
+
+	"github.com/openshift/library-go/pkg/operator/configobserver/featuregates"
+)
+
+func GetEnabledDisabledFeatures(features featuregates.FeatureGate) ([]string, []string) {
```

Comment:
Would appreciate some godoc describing this function and the other function

Reply:
Of course, I will push as soon as the initial test results arrive (don't want to terminate them now only to push comments)